### DISCUSSION TITLE
How can I create a smaller Magic Carpet

### DISCUSSION THREAD

**Discussion by Tygaran** [exult/exult#418] [General]
I'm been toying with the idea of attempting a remake of U5 in Exult.. Right now I'm gather ideas and feasibility of things.

One item of interest, is the magic carpet found near LB's room on the roof.  So I'm researching building a carpet with a smaller grid size... like 3x3 or 4x4 rather than the existing 12x6.  Another pierce of the puzzle is to retain the seatless style from the Keyring mod.

Interestingly, I can get a 6x6 magic carpet to fly by using both a single half of the carpet tiles along with the 'cushion' younormally use to (un)roll the carpet.  Issue here is that using the 'cushion' to roll and then unroll causes it to unroll into the original 12x6 version.  I'm guessing there's Usecode involved, but learning Usecode is a step further away on my list of things to learn.

What I'm looking for is a smaller grid, no seats, and the ability to place it nto inventory.  I do not need the (un)roll function, as jsu providing a proper paperdoll image should eb sufficient; and if the grid size is small enough it should fit into a backpack without obscuring everything else in there.

It'd likely be more usecode learning, but something similar was done for the balloon in the U6 mod... where the party actually disappears from view when the balloon lifts off.

Oh, that's another thing: is it possible to reduce the height at which a new magic carpet would fly?  The u5 one can be used inside after all, we just want it to fly over simple blocking terrain like swamps and not-too-tall terrain.

**Comment by KnightCaptainU7** [comment]
The U6 Mod source hasn't be released yet, to my knowledge.

Setting the Z axis of the carpet when used shouldn't be too hard.
